source/extensions/filters/http/gcp_events_convert/gcp_events_convert_filter.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <string>
+
+#include "envoy/server/filter_config.h"
+
+#include "common/common/logger.h"
+
+#include "envoy/extensions/filters/http/gcp_events_convert/v3/gcp_events_convert.pb.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace GcpEventsConvert {
+
+struct GcpEventsConvertFilterConfig {
+  GcpEventsConvertFilterConfig(const envoy::extensions::filters::http::gcp_events_convert::v3::GcpEventsConvert& proto_config);
+
+  const std::string key_;
+  const std::string val_;
+};
+
+using GcpEventsConvertFilterConfigSharedPtr = std::shared_ptr<GcpEventsConvertFilterConfig>;
+
+/**
+ * The filter instance for convert Cloud Event Pubsub Binding to HTTP binding
+ */
+class GcpEventsConvertFilter : public Http::StreamDecoderFilter , public Logger::Loggable<Logger::Id::filter>{
```

Comment:
Have you tried running the formatting script (https://github.com/YaronKoller/envoy/blob/master/support/README.md)? It should probably look like this:
```suggestion
class GcpEventsConvertFilter : public Http::StreamDecoderFilter, public Logger::Loggable<Logger::Id::filter> {
```

Reply:
I have trouble installing some dependency of that tool and I will try to figure that out. 
Thank you for the tip!